src/main/webapp/trip/creatTrip.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title>New Trip</title>
+         
+        <!-- Stylesheet --> 
+        <link rel="stylesheet" href="../style.css">
+
+        <!-- Fonts -->
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,400;0,500;0,600;0,700;0,800;1,400;1,500;1,600;1,700;1,800&display=swap" rel="stylesheet">
+
+        <!-- Favicon -->
+        <link rel="icon" href="images/favicon.ico">
+    </head>
+
+    <body>
+        <div class="container">
+            <h1 class="home-title">New Trip</h1>
+            <h3 class="home-subtitle">Start a new trip</h3>
+            <div class="enter-btn-div">
+                <form action="NewTrip" method="POST">
+                    <div class="inputBox">
+                        <label>Trip name </label></br>
+                        <input name="text-input-title" type="text" placeholder="" required="required">
+                        
+                    </div>
+                    <div class="inputBoxDate">
+                        <label>Date</label></br> 
+                        <label>from </label>
+                        <input name="text-input-start-date" type="date" placeholder="Start date" min="2022-01" max="2025-12" required="required">
+                        <label> to </label>
+                        <input name="text-input-end-date" type="date" placeholder="End date" min="2022-01" max="2025-12" required="required">
+                    </div>
+                    <div class="inputBox">
+                        <label>Total Budget in CAD </label></br>
```

Comment:
I think we can leave the currency out since we don't do anything with it and it's just for the user.
Ex. If a user inputted an amount in Yen vs USD it would still work as intended.